internal/worker/proxy_test.go

```diff
@@ -72,11 +74,24 @@ func (p *proxy) ServeHTTP(wr http.ResponseWriter, req *http.Request) {
 		appendHostToXForwardHeader(req.Header, clientIP)
 	}
 
+	bodyBytes, err := io.ReadAll(req.Body)
+	if err != nil {
+		http.Error(wr, "Cant' read request body: "+err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	bodyString := string(bodyBytes)
+
+	req.Body = io.NopCloser(strings.NewReader(bodyString))
+
 	resp, err := client.Do(req)
 	if err != nil {
 		http.Error(wr, "Server Error", http.StatusInternalServerError)
 	}
-	defer resp.Body.Close()
+	if resp.Body != nil {
+		defer resp.Body.Close()
+	}
+	p.paths = append(p.paths, strconv.Itoa(p.calls)+": "+req.URL.Path+" Body: "+strings.TrimSpace(bodyString)+" (Response: "+resp.Status+")")
```

Comment:
Nitpick, fmt.Sprintf would be cleaner I think.

Reply:
hmm - no idea why I didn't use Sprintf in the first place :-/